rakelib/pwsh.rake

```diff
@@ -219,6 +222,20 @@ namespace :pwsh do
             ruby_arg:                   'bare',
             validate_not_null_or_empty: true
           }
+        when 'module'
+          # bolt module install
+          # bolt module add [module]
+          @pwsh_command[:options] << {
+            name:                       'Module',
+            ruby_short:                 'md',
+            help_msg:                   'The name of the module to add to the Bolt project',
+            mandatory:                  false,
+            type:                       'string',
+            switch:                     false,
+            position:                   0,
+            ruby_arg:                   'bare',
+            validate_not_null_or_empty: true
+          }
```

Comment:
This will add `Module` as an optional parameter to all of the `BoltModule` cmdlets. This should be wrapped in a case/if so that it only applies to the `add` action and can be listed as a mandatory parameter.

Reply:
Ah, for other commands we just set it as not-mandatory if any subcommands didn't require it. I fixed those up as well.